packages/web-components/src/components/ic-stepper/ic-stepper.tsx

```diff
@@ -95,11 +95,16 @@ export class Stepper {
 
     if (this.alignedFullWidth) {
       this.stepperWidth = this.el.offsetWidth;
-      lastStep.style.maxWidth = `${this.stepperWidth / this.steps.length}px`;
+      lastStep.style.maxWidth = this.pxToRem(`${this.stepperWidth / this.steps.length}px`);
       this.lastStepWidth = lastStep.offsetWidth;
     }
   };
 
+  private pxToRem = (px: string, base = 16) => {
+    const tempPx = parseInt(px)
+    return (1 / base) * tempPx + 'rem'
+  }
```

Comment:
This method can be moved to `/src/utils/helpers.ts` so it can be used in other components if required. 
And can be updated slightly so it uses template literals instead.
```
`${((1 / base) * tempPx)rem`
```
(https://www.digitalocean.com/community/tutorials/understanding-template-literals-in-javascript - Look up Expression Interpolation)